src/main/java/com/marginallyclever/convenience/W3CColorNames.java

```diff
@@ -164,4 +169,16 @@ public class SVGColorNames {
     public static ColorRGB get(String name) {
         return map.get(name);
     }
+
+    /**
+     * @param match the color to matcch
+     * @return the name matching the given color, or null if no match.
+     */
+    public static String get(ColorRGB match) {
+        for(String key: map.keySet()) {
```

Comment:
you don't have to iterate through the map, you can use : `map.get(match)`

Reply:
And the method `isEqualsTo()` can be removed and the method `equals()` can be used in place.

Reply:
There is only map.get(key), no map.get(value).  I am doing the latter.

isEqualsTo() is a weird method.  removed.  ty!